lib/rails_admin/support/csv_converter.rb

```diff
@@ -32,6 +32,7 @@ def initialize(objects = [], schema = {})
     end
 
     def to_csv(options = {})
+      options = HashWithIndifferentAccess.new(options)
```

Comment:
I guess this is easier but I prefered to use the safest option. First, not knowing the rest of the code and the consequences, I prefered to be able to use string and symbol as keys, to be safe. Also, you suggestion would actually be to use `.deep_symbolize_keys!` 
At least using `HashWithIndifferentAccess`, I am sure I am not introducing any regression.